autoswap/autoswap_test.go

```diff
@@ -13,7 +13,9 @@ import (
 
 func getTestDb(t *testing.T) *database.Database {
 	db := &database.Database{
-		Path: ":memory:",
+		// its important to use a shared cache here since the sql driver can open up multiple connections
```

Comment:
```suggestion
		// it's important to use a shared cache here since the SQL driver can open up multiple connections
```
That' an important one :partying_face: